src/common/error.ts

```diff
@@ -20,7 +22,24 @@ export const createUserError =
       throw developerError('expected positive error code', 1);
     }
 
-    return new CLIError(emoji ? '❌ ' + message : message, { exit: code });
+    let final = message;
+
+    if (addSuffix) {
+      final =
+        final +
+        template(`\n\nIf you need help with this error, please raise an issue on GitHub Discussions: {underline.bold https://sfc.is/discussions}
+
+Please include the following information:
+command you ran
+error message
+version of the CLI, OS: superface cli/${VERSION} (${process.platform}-${process.arch}) ${process.release.name}-${process.version}
```

Comment:
nitpick: I'd add bullets here to make it visually more addressed

```suggestion
- command you ran
- error message
- version of the CLI, OS: superface cli/${VERSION} (${process.platform}-${process.arch}) ${process.release.name}-${process.version}
```